Mathlib/CategoryTheory/Yoneda.lean

```diff
@@ -990,6 +1085,40 @@ def compYonedaCompWhiskeringLeftMaxRight {D : Type u₂} [Category.{max v₁ v
   NatIso.ofComponents (fun X => hF.homNatIsoMaxRight _)
     (fun f => by ext; exact Equiv.ulift.injective (hF.map_injective (by simp)))
 
+/-- `FullyFaithful.homEquiv` as a natural isomorphism, using coyoneda. -/
+@[simps!]
+def homNatIso' {D : Type u₂} [Category.{v₂} D] {F : C ⥤ D} (hF : F.FullyFaithful) (X : C) :
+    F ⋙ coyoneda.obj (op (F.obj X)) ⋙ uliftFunctor.{v₁} ≅
+      coyoneda.obj (op X) ⋙ uliftFunctor.{v₂} :=
```

Comment:
Could you use `uliftCoyoneda` in the definition?
Also, I am not sure we absolutely need all the subsequent definitions with special values of the universes.